chapters/CRUD-Operations.md

```diff
@@ -207,612 +207,539 @@ Encode the **did:btcr2** identifier as follows:
+controller resolves to the contemporary DID document. That is the DID document
+that was current when the capability was invoked.
 
-* `targetDIDDocument` - The ::Contemporary DID Document:: with all updates applied to it; object.
+##### Hide {.unnumbered .unlisted}
 
-The steps are as follows:
+##### Imperative Algorithm {.unnumbered .unlisted}
 
-1. Set `capabilityId` to `update.proof.capability`.
-1. Set `rootCapability` to the result of passing `capabilityId` to the [Dereference Root Capability Identifier] algorithm.
-1. If `rootCapability.invocationTarget` does not equal `contemporaryDIDDocument.id` and `rootCapability.controller` 
-   does not equal  `contemporaryDIDDocument.id`, MUST throw an `invalidDidUpdate` error.
-1. Instantiate a [`bip340-jcs-2025` `cryptosuite`](https://dcdpr.github.io/data-integrity-schnorr-secp256k1/#instantiate-cryptosuite)
-   instance using the key referenced by the `verificationMethod` field in the update.
-   ```{.json include="json/CRUD-Operations/Update-apply-did-update.json"}
-   ```
-1. Set `expectedProofPurpose` to "capabilityInvocation".
-1. Set `mediaType` to "application/ld+json".
-1. Set `documentBytes` to the bytes representation of `update`.
-1. Set `verificationResult` to the result of passing `mediaType`, `documentBytes`,
-   `cryptosuite`, and `expectedProofPurpose` into the
-   [Verify Proof algorithm](https://w3c.github.io/vc-data-integrity/#verify-proof)
-   defined in the Verifiable Credentials (VC) Data Integrity specification.
-1. If `verificationResult.verified` equals False, MUST raise an `invalidUpdateProof` exception.
-1. Set `targetDIDDocument` to a copy of `contemporaryDIDDocument`.
-1. Use JSON Patch to apply the `update.patch` to the `targetDIDDOcument`.
-1. Verify that `targetDIDDocument` is conformant with the data model specified
-   by the DID Core specification.
-1. Set `targetHash` to the result of passing `targetDIDDocument` to the [JSON Canonicalization and Hash] algorithm.
-1. Check that `targetHash` equals the base58 decoded `update.targetHash`, else raise InvalidDidUpdate
-   error.
-1. Return `targetDIDDocument`.
+The algorithm takes in a did:btcr2 `identifier` and returns a `rootCapability`
+object.
+
+1. Define `rootCapability` as an empty object.
+2. Set `rootCapability.@context` to ‘https://w3id.org/zcap/v1’.
+3. Set `encodedIdentifier` to result of calling algorithm
+   `encodeURIComponent(identifier)`.
+4. Set `rootCapability.id` to `urn:zcap:root:${encodedIdentifier}`.
+5. Set `rootCapability.controller` to `identifier`.
+6. Set `rootCapability.invocationTarget` to `identifier`.
+7. Return `rootCapability`.
+
+#### Algo 13. Apply BTCR2 Update {.tabbed .unnumbered}
+
+This algorithm takes a ::BTCR2 Update::, attempts to verify it and then applies
+it to the appropriate DID document as identified by the update.
+
+A ::BTCR2 Update:: must contain the following:
+
+* @context. A context array containing the follow context URLs
```

Comment:
```suggestion
* @context: A context array containing the follow context URLs
```

Feel like some find and replace caught this. We should review and fix. All these should be colon : not .